src/utils/display-observer.js

```diff
@@ -178,7 +285,46 @@ export class DisplayObserver {
       const observations = this.targetObservations_.get(target);
       const oldDisplay = computeDisplay(observations, !this.isDocDisplay_);
       const newDisplay = computeDisplay(observations, this.isDocDisplay_);
```

Comment:
Is `computeDisplay` (the reducer part) still correct? Eg, If because the `body` observer displays the element but the `container` observer does not, then we're not really displayed. Right?

Reply:
So far, I think, for all cases I tried, I think it's ok to do a plain OR. But, yeah, something like this can happen. Or do you mean that a container-vs-body are mutually exclusive situation?